mono/utils/monobitset.h

```diff
@@ -9,26 +9,15 @@
 #include <mono/utils/mono-publib.h>
 #endif
 
-/*
```

Comment:
I think this change is big enough that it should be in a different commit. Also why add it both to publib and compiler headers?